File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpiritTome.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class SpiritTome extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_spirit_tome_name;
        this.idDescription = R.string.accessory_spirit_tome_description;
        this.idImage = R.drawable.spirit_tome;
        this.source.add(Integer.valueOf(R.string.dungeon_name_eternal_battlefield));
        this.price = 26L;
        this.intelligence = 9;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpiritTome.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class SpiritTome : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_spirit_tome_name
        idDescription = R.string.accessory_spirit_tome_description
        idImage = R.drawable.spirit_tome
        source.add(R.string.dungeon_name_eternal_battlefield)
        price = 26L
        intelligence = 9
    }
}
